### PR TITLE
feat<FM-256>: Create Reusable Button for Level Balloons -> Implemented reusable buttons for rendering balloons in level selection and refactor code paths especially level selection scene.

### PR DESCRIPTION
**Pull request by bernhardccodev**
Feat:  Create Reusable Button for Level Balloons

# Changes

- Deleted old component/level-config.ts.
- Create a new level-config.ts that includes handling of images and special images depending on the level and moved necessary values for level-config from level-selection-scene.ts
- Created a reusable button for levels called component/buttons/level-bloon-button.ts.
- Optimized the rendering of button levels by moving all logic to component/buttons/level-bloon-button.ts
- Code clean up for composition folder.

# How to test
- Run FTM
- Click Play on start scene.
- In level-selection-scene. button should have the visual effect when  pressed down and all levels should properly render including the stars and locks as well as load the correct game level.

Ref: [JIRA-CODE](https://curiouslearning.atlassian.net/browse/FM-256)

### CODE REVIEW

File: src/compositions/level-config.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        [
+            canvas.width / 2.5,
+            canvas.height / 1.3
+        ],
+      ],
+    ];
+}
+
+export const loadLevelImages = async () => {
+    return await syncLoadingImages(
```
Comment (woodsjohn-codev) [inline src/compositions/level-config.ts:61]: @bernhardccodev i think theres no need to await here since the caller can handle the Promise.

**Review by woodsjohn-codev — COMMENTED** [review]
Looks good to me so far. only one tiny comment.

File: build/feedTheMonster.js
Comment (ashwinnair-chimple) [inline build/feedTheMonster.js:1]: try avoiding the build folder in commit to prevent merge conflicts

File: src/components/buttons/index.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import PauseButton from "./pause-button";
 import RetryButton from "./retry-button";
 import YesButton from "./yes-button";
 import PlayButton from "./play-button";
+import LevelBloonButton from './level-bloon-button';
```
Comment (ashwinnair-chimple) [inline src/components/buttons/index.ts:9]: There is a spelling mistake for balloon

File: src/components/buttons/level-bloon-button.ts
Comment (ashwinnair-chimple) [inline src/components/buttons/level-bloon-button.ts:1]: Change File name

File: src/components/buttons/index.ts (hunk 2 above)
Comment (bernhardccodev) [reply src/components/buttons/index.ts:9]: The bloon is short for balloon word. It's a pun or word play.

**Review by ashwinnair-chimple — APPROVED** [review]
Approved and can be merged to dev-optimization branch

**Pull request merged**